print-numbers-server/index.js

```diff
@@ -0,0 +1,20 @@
+//Call the http module and define several constants (http,host,port) 
+const HTTP = require("http");

```

Comment:
http is an object instance with no static constant value. Because we don't know what is the exact value.

Reply:
Error corrected